flexget/components/managed_lists/lists/radarr_list.py

```diff
@@ -393,6 +416,26 @@ def items(self):
             self._movie_entries = self._get_movie_entries()
         return self._movie_entries
 
+    @property
+    def tags(self):
+        """ Property that returns tag by id """
+        if not self.config_tags:
+            self._tags = []
+            return self._tags
+
+        tags_ids = []
+        if self._tags is None:
+            existing = {t["label"].lower(): t["id"] for t in self.service.get_tags()}
+            for tag in self.config_tags:
+                tag = tag.lower()
+                found = existing.get(tag)
+                if not found:
+                    logger.verbose('Adding missing tag %s to Radarr' % tag)
+                    found = self.service.add_tag(tag)["id"]
```

Comment:
this seems counter-intuitive to occur when querying a class property. Maybe add this logic in a setter?